dkg/dkg.go

```diff
@@ -219,18 +219,15 @@ func setupP2P(ctx context.Context, key *ecdsa.PrivateKey, p2pConf p2p.Config, pe
 		return nil, nil, errors.Wrap(err, "")
 	}
 
-	relays, err := p2p.NewRelays(p2pConf, bootnodes)
-	if err != nil {
-		return nil, nil, err
-	}
+	relays := p2p.NewRelays(p2pConf, bootnodes)
 
 	tcpNode, err := p2p.NewTCPNode(p2pConf, key, p2p.NewOpenGater(), udpNode, peers, relays)
 	if err != nil {
 		return nil, nil, errors.Wrap(err, "")
 	}
 
 	for _, relay := range relays {
-		go func(relay p2p.Peer) {
+		go func(relay *p2p.MutablePeer) {
```

Comment:
nit: can we not have a pointer here?

Reply:
what do you mean?

Reply:
it is safe since the value of the pointer is copied